assets/css/admin.css

```diff
@@ -130,6 +130,16 @@
 	background-size: contain;
 }
 
+.payment-method__brand--card_present {
+	background: no-repeat url( '../images/cards/card-present.svg' );
```

Comment:
Thanks @kalessil. Do you mean that we should display the brand information of the card present payment, like for instance show the visa logo if it was paid with a visa?

The rest of regular payments build the CSS class based on the transaction info in the server cache i.e a transaction paid with a visa card has `visa` as `source` and the generated CSS is `payment-method__brand--visa`.

In the case of card present payments we don't have the brand information in the cache, the source is stored like 'card_present'. 

The `brand` attribute is available in the charge information, but we would need an additional API call to retrieve it, as far as I understand, and would have impact in the performance. 

I guess other option would be to cache the brand information in the server but not sure if this is worth it.

Thinking on an alternative approach, we could use a generic icon for this type of payments (a card reader or something like that). Maybe this could work, taking into account that currently we are displaying the correct logo in the details of the charge for card present payments. So in the list the merchant would identify the payment in the list as in person payment and could check the specific brand clicking in the table row.

I'd appreciate your comments since I am not sure how to proceed. Maybe I am missing something?

Reply:
> Do you mean that we should display the brand information of the card-present payment, like for instance show the visa logo if it was paid with a visa?

Correct. Or Mastercard or another brand.

> I guess another option would be to cache the brand information in the server but not sure if this is worth it.

I can see that IPP transactions have the brand name in there, similar to online card payments.
And the online card payments brand is recognized. 

So it' s either:
- server is not sending the IPP branding information
- the client is not using IPP branding information sent by the server
- both the server/client needs to be updated for properly using  IPP branding information
Which case we are dealing with here?